cmd/agent/app/integrations.go

```diff
@@ -238,26 +240,26 @@ func pip(args []string) error {
 	pipCmd := exec.Command(pythonPath, args...)
 
 	// forward the standard output to stdout
-	stdout, err := pipCmd.StdoutPipe()
+	pip_stdout, err := pipCmd.StdoutPipe()
```

Comment:
Go convention is to use mixedCaps https://golang.org/doc/effective_go.html#mixed-caps for variables name (surprised `go vet` didn't raised an issue).

Could you update the PR to follow the convention ?

Reply:
Sure thing 👍